### DISCUSSION TITLE
海外哪个地区，阿里网盘能用？

### DISCUSSION THREAD

**Discussion by skrik2** [AlistGo/alist#7009] [General]
https://github.com/alist-org/alist/discussions/6874

之前用 alist 的阿里网盘，上传速度慢的要死

这几天，整了一个远程桌面

在新加坡的服务器上，安装阿里网盘客户端，上传速度仍然慢的一批

有什么海外地区是正常的

**Comment by anwen-anyi** [comment]
不支持非大陆地区以外地区

**Reply by skrik2** [reply]
我就说，这玩意怎么上传东西这么卡

**Comment by skrik2** [comment]
更新一下，新加坡能正常上传了
